src/commands/register.ts

```diff
@@ -56,6 +60,14 @@ export default class ComponentRegister extends BaseCommand {
         description: 'Directory to write build cache to. Do not use in Github Actions: https://docs.architect.io/deployments/automated-previews/#caching-between-workflow-runs',
       }),
     },
+    environment: {
```

Comment:
should we make this `hidden: true`?

Reply:
I had considered that, but I don't see why someone shouldn't be able to create an ephemeral component version directly rather than indirectly thorough the deploy command

Reply:
What would be the reason for them to do that? My inclination is to make it hidden with the option to unhide in the future.

Reply:
Maybe they just don't want to deploy the component immediately after it's registered? I'll hide it though